scipy/stats/tests/test_distributions.py

```diff
@@ -7284,6 +7284,23 @@ def test_sf_isf(self):
         x1 = stats.nakagami.isf(sf, nu)
         assert_allclose(x1, x0, rtol=1e-13)
 
+    @pytest.mark.parametrize("m, ref",
+    [(5, -0.097341814372152004341365781403404039274592838861342746),
+    (0.5, 0.72579135264472741361871915897806499359677157053889381),
+    (10, -0.43426184310934906286934353114924211110708634428061666)])
```

Comment:
Could you run tests with very small and very large values for `m`?

Reply:
>Could you run tests with very small and very large values for m?

Not really. My mpmath implementation throws an error when I try to use an m larger than 100. That being said, I did change the implementation of ```_entropy``` so it doesn't overflow, so testing for accuracy at extreme values should be unnecessary.

Reply:
In my experience, extreme values are where formulas such as the one you are implementing here, are most valuable as they are much more accurate than numerical integration. Could you have a look at my mpmath implementation below and add one test for very small `m` and one for very big `m`?

Reply:
I agree with @dschmitz89 here. You'd find that at least the test with large `m` will fail because of the issue mentioned above (https://github.com/scipy/scipy/pull/17868#discussion_r1089670054).